sdk/storageactions/azure-resourcemanager-storageactions/src/test/java/com/azure/resourcemanager/storageactions/StorageActionsManagerTests.java

```diff
@@ -0,0 +1,121 @@
+import com.azure.resourcemanager.storageactions.models.ManagedServiceIdentity;
+import com.azure.resourcemanager.storageactions.models.ManagedServiceIdentityType;
+import com.azure.resourcemanager.storageactions.models.OnFailure;
+import com.azure.resourcemanager.storageactions.models.OnSuccess;
+import com.azure.resourcemanager.storageactions.models.StorageTask;
+import com.azure.resourcemanager.storageactions.models.StorageTaskAction;
+import com.azure.resourcemanager.storageactions.models.StorageTaskOperation;
+import com.azure.resourcemanager.storageactions.models.StorageTaskOperationName;
+import com.azure.resourcemanager.storageactions.models.StorageTaskProperties;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Random;
+
+public class StorageActionsManagerTests extends TestProxyTestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_WEST3;
+    private String resourceGroupName = "rg" + randomPadding();
+    private StorageActionsManager storageActionsManager = null;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new AzurePowerShellCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        resourceManager = ResourceManager.configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        storageActionsManager = StorageActionsManager.configure()
+            .withPolicy(new ProviderRegistrationPolicy(resourceManager))
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups().define(resourceGroupName).withRegion(REGION).create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @LiveOnly
+    public void testCreateStorageTask() {
+        StorageTask storageTask = null;
+        try {
+            String taskName = "task" + randomPadding();
+            Map<String, String> operationMap = new LinkedHashMap<>();
+            operationMap.put("tier", "Hot");
+            // @embedmeStart
```

Comment:
put this before `Map<String, String> operationMap = new LinkedHashMap<>();`

Reply:
Fixed by the [commit#a545222](https://github.com/Azure/azure-sdk-for-java/pull/45208/commits/a545222e11e6df65bb1ec2ef52448b69b9c143df)